tests/test_fips.py

```diff
@@ -320,3 +320,35 @@ def test_icainfo_binary(container_per_test: ContainerData) -> None:
     )
     container_per_test.connection.check_output("icastats -k")
     container_per_test.connection.check_output("icastats -S")
+
+
+@pytest.mark.parametrize(
+    "container_per_test", FIPS_TESTER_IMAGES, indirect=True
+)
+def test_nss_firefox_cert(container_per_test: ContainerData) -> None:
+    container_per_test.connection.check_output(
+        "zypper -n install mozilla-nss mozilla-nss-tools"
```

Comment:
Why is `mozilla-nss` needed?

Reply:
It's a requirement of `mozilla-nss-tools` anyway...

```01fe315846a8:/ # zypper -n install mozilla-nss-tools
Refreshing service 'container-suseconnect-zypp'.
Retrieving repository 'SLE_BCI' metadata ................................................................[done]
Building repository 'SLE_BCI' cache .....................................................................[done]
Loading repository data...
Reading installed packages...
Resolving package dependencies...

The following 6 NEW packages are going to be installed:
  libfreebl3 libsoftokn3 mozilla-nspr mozilla-nss mozilla-nss-certs mozilla-nss-tools

The following 6 packages are not supported by their vendor:
  libfreebl3 libsoftokn3 mozilla-nspr mozilla-nss mozilla-nss-certs mozilla-nss-tools

6 new packages to install.

Package download size:     2.5 MiB

Package install size change:
              |       6.3 MiB  required by packages that will be installed
     6.3 MiB  |  -      0 B    released by packages that will be removed
```

Reply:
I removed the `mozilla-nss` from the `zypper in ...` command and leave it be installed as required by `mozilla-nss-tools`

Reply:
Ok, so it doesn't really make a difference.